Dockerfile

```
@@ -0,0 +1,63 @@
+# Copy source code
+COPY athf/ ./athf/
+
+# Install ATHF in editable mode with all optional dependencies
+# This includes:
+# - Core dependencies (click, pyyaml, rich, jinja2)
+# - Similarity search (scikit-learn)
+# - Splunk integration (requests)
+RUN pip install --no-cache-dir -e ".[similarity,splunk]"
+
+# Create non-root container user
+RUN useradd -m -s /bin/bash athf
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)